package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-lineto",
-  "version": "3.2.1",
+  "version": "3.2.2",
```

Comment:
Mind omitting this change? I'll bump the version when I publish the next release to npm.

Reply:
It's okay for me to omit the version change. Just force pushed the changes. No version change now. Thanks for your review.